Week3/homework/step2-2.js

```diff
@@ -3,18 +3,28 @@
 function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
   const numbers = [];
 
-  // Replace this comment and the next line with your code
-  console.log(startIndex, stopIndex, threeCallback, fiveCallback, numbers);
+  for (let i = 0; i <= stopIndex - startIndex; i++) {
+    numbers.push(startIndex + i);
+  }
+
+  for (const number of numbers) {
+    if (!(number % 3)) {
```

Comment:
I guess you could read this as "if the number divided by 3 doesn't have a remainder ...", which you could take to mean "if the number is divisible by 3 ...". To me, the expression `if (number % 3 === 0)` more clearly expresses what is intended here.